target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -313,6 +313,12 @@ rp-n53)
 	ucidef_set_led_netdev "eth" "Network" "$board:white:back" "eth0"
 	set_wifi_led "$board:blue:wifi"
 	;;
+rt-ac51u)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.1
+	ucidef_set_led_netdev "activity" "activity" "$board:blue:activity" eth0.2
+	ucidef_set_wifi_led "$board:blue:activity"
```

Comment:
I'm quite sure that doesn't work.

The contents of ```/sys/devices/platform/gpio-leds/leds/rt-ac51u:blue:activity/device_name``` is eth0.1, will be changed to eth0.2 and should be wlan0 in the end. As far as I know the netdev trigger doesn't have support for "monitoring" multiple devices (```echo "eth0.1 eth0.2 wlan0" > /sys/devices/platform/gpio-leds/leds/rt-ac51u:blue:activity/device_name```).

You have to decide for one netdev here.